src/pages/Login/FBLogin.js

```diff
@@ -49,10 +44,32 @@ const FBLogin = ({ isDisabled, event, role }) => {
             },
             body: JSON.stringify(newProfile)
           })
+            .then(response => response.json())
             .then(function(response) {
-              // console.log(response)
+              // Armazena os dados do usuário no navegador. Quando houver logout eles serão apagados
+              localStorage.setItem('userId', response.id)
+              localStorage.setItem('userPicture', response.userPicture)
+              localStorage.setItem('userName', response.name)
+              localStorage.setItem('userEmail', response.email)
+              localStorage.setItem('userRole', '')
+
+              // Se ele já tinha escolhido o evento anteriormente, será redirecionado para lá após login
+              event? history.push(`/events/${event}`) :
```

Comment:
acho que essa validacao seria algo como
`!user.role ? history.push('/welcome') : history.push('/')`
ou se o event for validacao necessaria de outro fluxo
`!user.role ? history.push('/welcome') : (event ? history.push('/events/${event}') : history.push('/'))`